ROADMAP.md

```diff
@@ -0,0 +1,22 @@
+# Our Roadmap
+
+Hey, there. Thanks so much for the interest in openSNP and what we're trying to accomplish over the next weeks and months.
+
+We have some ongoing efforts which need to be done as well as some bug fixes and new features we'd love to implement. If you think we should focus on something else: Please just comment, add it or talk to us, whatever you feel like doing!
+
+
+## Ongoing Efforts
```

Comment:
This looks like a good breakdown to me! It looks a bit like I can't start working on the 'near future' issues just yet -- is that true?

This is also a good place to mention *why* you want to accomplish some of these tasks -- it helps cast vision around your project!